src/components/animated/_transparentOnHover.less

```diff
@@ -0,0 +1,107 @@
+.transparent-btn {
+  position: relative;
+  transition: 0.3s ease-in-out;
+  &:hover {
+    .bc(transparent);
+
+    &.blue-btn {
+      .textColorChange(@blue);
+    }
+    &.pink-btn {
+      .textColorChange(@pink);
+    }
+
+    &.green-btn {
+      .textColorChange(@green);
+    }
+
+    &.yellow-btn {
+      .textColorChange(@yellow);
+      &:hover {
+        color: @darkText !important;
+      }
+    }
+
+    &.orange-btn {
+      .textColorChange(@orange);
+    }
+
+    &.red-btn {
+      .textColorChange(@red);
+    }
+
+    &.purple-btn {
+      .textColorChange(@purple);
+    }
+
+    &.black-btn {
+      .textColorChange(@black);
+    }
+
+    &.white-btn {
+      .textColorChange(@white);
+      border-color: @whiteBorder;
+      &:hover {
+        color: grey !important;
```

Comment:
change color on hover to white

Reply:
In white it will not be visible

Reply:
It will not be visible on the website but that's the functionality of the button if someone were to use it.